Content.Server/Power/EntitySystems/CableSystem.cs

```diff
@@ -69,7 +69,7 @@ private void OnAnchorChanged(EntityUid uid, CableComponent cable, ref AnchorStat
 
         // anchor state can change as a result of deletion (detach to null).
         // We don't want to spawn an entity when deleted.
-        if (!TryLifeStage(uid, out var life) || life >= EntityLifeStage.Terminating)
+        if (LifeStage(uid) >= EntityLifeStage.Terminating)
```

Comment:
```suggestion
        if (TerminatingOrDeleted(uid))
```
just use the method for this